synfig-studio/src/synfigapp/vectorizer/outlinevectorizer.cpp

```diff
@@ -1,52 +0,0 @@
-/* === S Y N F I G ========================================================= */
```

Comment:
The build fails because of missing file - synfig-studio/src/synfigapp/vectorizer/outlinevectorizer.cpp
I guess you need  to edit `Makefile.am` and `CMakeLists.txt`. ^__^